src/core/mainloop/mainloop.c

```diff
@@ -1959,6 +1959,11 @@ write_stats_file_callback(time_t now, const or_options_t *options)
     if (next_write && next_write < next_time_to_write_stats_files)
       next_time_to_write_stats_files = next_write;
   }
+  if ( !!options->BandwidthStatistics) {
```

Comment:
Please explain that we collect statistics when the option is true or auto.
(Otherwise, someone might forget that the option is an autobool, and remove the !!)